Example/Tests/Tests.m

```diff
@@ -53,24 +53,25 @@ - (SCORStreamingAnalytics *)create;
     __block Class scorAnalyticsClassMock;
     __block SCORStreamingAnalytics *mockStreamingAnalytics;
     __block SEGComScoreIntegration *integration;
-    __block SCORStreamingPlaybackSession *mockPlaybackSession;
-    __block SCORStreamingAsset *mockAsset;
+    __block SCORStreamingConfiguration *mockConfiguration;
+//    __block SCORStreamingPlaybackSession *mockPlaybackSession;
```

Comment:
Should these lines be commented out or just removed?

Reply:
I will remove those, sorry about that.